src/main/proto/th2_grpc_common/common.proto

```diff
@@ -42,6 +42,14 @@ message MessageID {
   repeated uint32 subsequence = 4; // List of nested sequences in addition to the main one
 }
 
+message EventBatchMetadata {
+  string external_user_queue = 1; // External user queue can be used for routing by internal box logic.
```

Comment:
Maybe rename `response_queue`?

Reply:
I'd save external. Because this queue isn't under infra control